code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -482,6 +488,7 @@
 /datum/reagent/toxin/cyanide
 	name = "Cyanide"
 	id = "cyanide"
+	value = 1 //Also usefull in labs
```

Comment:
IRL usefull in chem labs

Reply:
So is ammonia, and basic buffer, and acidic buffer, aswell as basically everything else that the machines dispense...

Reply:
Cyanide is used in gold leaching and recycling, its a usefull /toxin/. Its going to sell better then buffes that are easy to make